### DISCUSSION TITLE
Error trying to run exampleSite

### DISCUSSION THREAD

**Discussion by Masqued** [pacollins/calligraphy#16] [Q&A]
I just installed this lovely theme via git. When I run the server, I get an error that the contact-form shortcode is not found. Here is the exact error (other than path):

`Error: error building site: process: readAndProcessContent: "/path/to/newsite/themes/calligraphy/exampleSite/content/contact/index.md:8:1": failed to extract shortcode: template for shortcode "contact-form" not found`

Using hugo v0.136.3+extended darwin/amd64 on Mac.

It sounds to me like somehow the server isn't locating the shortcode's file. Any suggestion would be appreciated.

**Comment by pacollins** [comment]
It looks like you are trying to build the example site (which builds on Netlify without error). To do this, either need to put the theme files within the example site directory, or define the directory as up two.

`hugo --themesDir ../../`

I think that is the correct flag and value.

Option B is to move all the `exampleSite` files to `newsite`.

**Reply by Masqued** [reply]
Thanks for your reply. I have tried both, and it is still giving the same error. I'll keep working on it.

**Reply by lgmg** [reply]
Thank you for this template. I have the same issue. One need to remove the contact form short-code if executed from localhost and net from netlify. But still have an issue on the exampleSite because there is no home generated. (perhaps no index.html in the layout)
